Extensions/ParticleSystem/Extension.cpp

```diff
@@ -102,6 +102,9 @@ ParticleSystemCppExtension::ParticleSystemCppExtension() {
   conditions["ParticleSystem::EmitterAngle"]
       .SetFunctionName("GetAngle")
       .SetIncludeFile("ParticleSystem/ParticleEmitterObject.h");
+  conditions["ParticleSystem::ParticleType"]
+      .SetFunctionName("GetRendererType")
+      .SetIncludeFile("ParticleSystem/ParticleEmitterObject.h");
```

Comment:
Nice to see that you implemented it for the C++ version, but as it's getting deprecated I suggest we don't add this and keep the implementation only for the JS engine :)